utils.py

```diff
@@ -37,8 +37,14 @@ def check_roles(user: discord.Member, valid_roles: list[int]) -> bool:
             return True
     return False
 
-# Since usernames can have spaces, first check if it's a username, otherwise just cut off first word as normal
-def parse_message(message: str, username: str) -> str:
+# Removes a user mention from a message
+def parse_message(message: str, username: str, username_in_message: bool) -> str:
```

Comment:
We'll either need to make a change to Governor to reflect this change in API, or possibly create some thin wrapper function so Governor can use the version with the default setting (if there is a globally acceptable default setting)

Reply:
Does Governor use this API?

I didn't see any references.

Defaulting to true makes sense anyway, I'll add that.

Reply:
...I may not have actually checked if it did or not lol.